### ISSUE TITLE
Update Travis-CI example in README

### ISSUE THREAD

**Issue by SethMMorton** [joerick/cibuildwheel#223]
First, I want to say that part of the reason I am opening this issue is to say how awesome this project is. Thank you for doing this!

---

There are some things that I would recommend changing in the Travis-CI example

- `sudo: required` is no longer needed, and is requested to be omitted now (https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration)
- The whole `PIP=` things can be omitted if calls to `pip` use `python -m pip`
- Installation of dependencies could be moved to a `install` step
- Instead of setting an environment variable with the user name, it could be added to the `twine` call (this is just a preference)
- Windows could be added

It could be updated to look like

```yaml
language: python

matrix:
  include:
    # perform a linux build
    - services: docker
    # and a mac build
    - os: osx
      language: generic
    # and a windows build
    - os: windows
      language: generic
      before_install:
        - choco install python --version 3.8.0
        - export PATH="/c/Python38:/c/Python/Scripts:$PATH"

install:
  - python -m pip -U pip
  - python -m install twine cibuildwheel==1.0.0

script:
  # build the wheels, put them into './wheelhouse'
  - python -m cibuildwheel --output-dir wheelhouse
  # if the release was tagged, upload them to PyPI
  - |
    if [[ $TRAVIS_TAG ]]; then
      python -m twine upload --user joerick wheelhouse/*.whl
      # Note: TWINE_PASSWORD is set in Travis settings
    fi
```

There are even ways to make `cibuildwheel` get called in a separate "deploy" stage from one's tests, which is quite nice. That could be added as an alternate example, but then I suppose you support multiple services and you don't want to seem like you favor Travis too much.

I'd be happy to make a PR if you are interested.

**Comment by YannickJadoul** [comment]
Thanks for these suggestions, @SethMMorton. Except for adding Windows, none of these are really high on the list of things to urgently do, but ... yeah, I'm OK with these; it makes things a little bit cleaner indeed :-)

I expect others to quick like the cleanup as well. Do you have time to make a PR? If not, I can do it.
And if you feel like it, feel free to have a look at the other examples in the as well: https://cibuildwheel.readthedocs.io/en/latest/setup/

One thing I'm a bit concerned about: are these actually tested? Are these part of our test suite, somehow, @joerick, @mayeut? And should they?

**Comment by mayeut** [comment]
Seems cleaner, thanks.

Maybe a PR over at https://github.com/joerick/cibuildwheel-autopypi-example to check everything's working properly then updating the docs ?

**Comment by SethMMorton** [comment]
I'd be happy to make a PR to both repositories.  I'll limit the README changes to the Windows addition, removal of `sudo` (since the Travis folks would like that going forward), and the removal of the `PIP` envvar (to reduce complexity).

I have not tested *this* example, but I lifted the code from one of my repos and it works - though I agree that it is best to actually test the real example before giving it over to people!

**Comment by SethMMorton** [comment]
OK, I made a PR at https://github.com/joerick/cibuildwheel-autopypi-example/pull/5. In that PR is the link to the TravisCI build. Let me know what you think of the changes there, and I can apply those changes to the PR I will make here.

**Comment by joerick** [comment]
Fixed in #228
